src/assets/styles/vuetifyTheme.js

```diff
@@ -103,33 +103,33 @@ export default {
       },
       dark: {
       // new colors
-        backgroundWallet: '#F2F4FA',
-        backgroundOverlay: '#F2FAFA',
-        backgroundGrey: '#F8F9FB',
+        backgroundWallet: '#000000',
+        backgroundOverlay: '#000000',
+        backgroundGrey: '#000000',
         whiteAlways: '#FFFFFF',
-        whiteBackground: '#FFFFFF',
-        whitePopup: '#FFFFFF',
-        textDark: '#192133',
-        textMedium: '#5A678A',
-        textLight: '#939FB9',
-        greyPrimary:'#5A678A',
-        greyMedium: '#D7DAE3',
-        greyLight: '#F2F3F6',
-        greenPrimary: '#05C0A5',
-        greenMedium: '#C3F0E9',
-        greenLight: '#EBFAF8',
-        bluePrimary: '#4B83E8',
-        blueMedium: '#D4E1F9',
-        blueLight: '#EEF3FD',
-        redPrimary: '#FF445B',
-        redMedium: '#FFD2D8',
-        redLight: '#FFF0F2',
-        orangePrimary: '#F5A623',
-        orangeMedium: '#FBDBA7',
-        orangeLight: '#FEF4E5',
-        disabledPrimary: '#CECECE',
-        disabledMedium: '#E0E0E0',
-        disabledLight: '#F6F6F6',
+        whiteBackground: '#192439',
+        whitePopup: '#202b43',
+        textDark: '#ccd4e6',
+        textMedium: '#626e86',
+        textLight: '#3a4562',
+        greyPrimary:'#5a678a',
+        greyMedium: '#323d57',
+        greyLight: '#26324a',
+        greenPrimary: '#06d9b9',
+        greenMedium: '#107e7a',
+        greenLight: '#154853',
+        bluePrimary: '#6292eb',
+        blueMedium: '#355288',
+        blueLight: '#283a5d',
+        redPrimary: '#ff5d70',
+        redMedium: '#8c4155',
```

Comment:
redMedium is #FF5E70

Reply:
Fixed

Reply:
I think you changed redPrimary and not red medium

Reply:
![Screenshot from 2021-12-21 16-14-45](https://user-images.githubusercontent.com/9893774/147013854-088378a6-66a6-468a-98c2-5dddd3be385a.png)
![Screenshot from 2021-12-21 16-14-50](https://user-images.githubusercontent.com/9893774/147013855-f13d0833-2286-494a-b2c2-98327abe0db2.png)

Same thing. I capture the real redMedium value from chrome debug pannel, so it should be fine.